libraries.json

```diff
@@ -1118,5 +1118,10 @@
     "github": "MyUNiDAYS/couchbaselite-kotlin-sdk",
     "category": "Storage",
     "maven": "https://repo1.maven.org/maven2/com/myunidays/couchbaselite/"
-  }
+  },
+    {
+    "github": "GuilhE/KMP-ComposeUIViewController",
+    "category": "UI",
+    "maven": "https://github.com/GuilhE/KMP-ComposeUIViewController/"
```

Comment:
Here should be path to maven repo with compiled library

Reply:
done